docs/index.html

```diff
@@ -341,6 +341,84 @@ <h2>Terminology</h2>
           <dfn data-lt="URI|URIs">URI</dfn>
         </dt>
         <dd>An identifier as defined by [[RFC3986]].</dd>
+        <dt>
+          <dfn data-lt="PATH|PATHs">PATH</dfn>
+        </dt>
+        <dd>
+          A distinct path for a distinct set of items being traced that either is declared
+          in advance by a user or system, or a root path that has already occurred
+          (in whole or in part). A PATH might branch, combine with others, or
```

Comment:
> A PATH might branch, combine with others, or terminate prematurely.

I feel like this is conflating a graph / tree, with the actual concept of a path (as in shortest path) in such a strucure.